.github/workflows/_deploy-hl7-notification-cdk.yml

```
@@ -0,0 +1,181 @@
+    # TODO Consider the solution here: https://github.com/tj-actions/aws-cdk/blob/main/action.yml
+    concurrency:
+      group: ${{ format('{0}-{1}', inputs.hl7_notification_stack, inputs.deploy_env) }}
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.deploy_env }}
+    steps:
+      - name: Log Environment
+        run: |
+          env
+        shell: bash
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)